data/mil2525d/utilities/military-features-utilities/README.md

```diff
@@ -3,5 +3,71 @@
 
 ## Purpose
 
-* These utilities are used to create military feature classes, domains, and other feature and test data   
-	* [update-domain-toolbox](./update-domain-toolbox) -  GP Tools for creating geodatabase domains for military feature classes
+* These utilities are used to create military feature classes, domains, and other feature and test data 
+* Individual utilities and steps:
+    * Update Military Features Domains - updates the geodatabase domains from source CSV(.csv) files
+
+## Sections
+
+* [Requirements](#requirements)
+* [Instructions](#instructions)
+
+## Requirements
+
+* ArcGIS Professional 1.0+ (Toolbox requires ArcGIS Pro to open/use)
+
+## Instructions
+
+### Update Military Features Domains
+
+#### Overview
+
+This utility updates the Geodatabase(GDB) domains of the [Military Features template database](../../core_data/gdbs) with the latest source data/values obtained from the [Joint Military Symbology](https://github.com/Esri/joint-military-symbology-xml) repository. This utility should be run periodically as the Joint Military Symbology repository is refined, improved, and updated.
+
+The source data for this utility is a set of CSV files created in the Joint Military Symbology [`name_domains_values folder`](https://github.com/Esri/joint-military-symbology-xml/tree/master/samples/name_domains_values). The domain name is obtained from the CSV file name (with the "Coded_Domain" part removed) and the domain codes and description are obtained from the file contents.
+
+A Geoprocessing (GP) Tool is then run on the source data to add or replace the GDB domains using the source data.
+
+As a final (optional) validation step, once the domain data is imported into the GDB, the domains are then exported and compared to the original source data.
+
+#### Steps
+
+Importing the domain data:
+
+* Obtain the latest set of Military Features source data and utilities
+    * Clone/download this repository to your local machine.
+* Obtain the latest set of `name_domains_values` source CSV files
+    * Clone/download the  [Joint Military Symbology](https://github.com/Esri/joint-military-symbology-xml) repository.
+    * You may also just download the files from the [`name_domains_values folder`](https://github.com/Esri/joint-military-symbology-xml/tree/master/samples/name_domains_values) in this repository
+    * Note: although there are addition sample CSV files in this folder (ex. `*_Sample.csv`), the tools ignore these files using a file filter
+* **REQUIRED WORKAROUND**
```

Comment:
Linking to https://github.com/Esri/joint-military-symbology-xml/issues/201 & CC'ing @abouffard so he is aware of this required workaround